config/config.go

```diff
@@ -170,6 +170,11 @@ type SystemConfiguration struct {
 
 		Uid int `yaml:"uid"`
 		Gid int `yaml:"gid"`
+
+		// Passwd controls weather a passwd file is mounted in the container
+		// at /etc/passwd to resolve missing user issues
+		Passwd     bool   `json:"mount_passwd" yaml:"mount_passwd" default:"true"`
+		PasswdFile string `json:"passwd_file" yaml:"passwd_file" default:"/etc/pterodactyl/passwd"`
```

Comment:
/etc/pelican

Reply:
I forgot about that change. I just copy/pasted my changes over.